book/src/internals/type-system.md

```diff
@@ -0,0 +1,224 @@
+# Type system
+
+> For variables, the type system determines the allowed values of that term.
+>
+> -- Wikipedia
+
+## Purpose
+
+Each of the SQL DBMSs has their own type system. Thanks to SQL standard, they
+are very similar, but have key differences. For example, SQLite does not have a
+type for date or time or timestamps, but it has functions for handling date and
+time that take ISO8603 strings or ints representing Unix timestamps. So it does
```

Comment:
ISO 8603 is about Solid Fertilizers: https://www.iso.org/standard/15908.html

I think you meant ISO 8601 😆

Reply:
```suggestion
time that take ISO 8601 strings or ints representing Unix timestamps. So it does
```